code/modules/vehicles/mecha/combat/greyscale/greyscale.dm

```diff
@@ -40,6 +40,9 @@
 	///right particle smoke holder
 	var/obj/effect/abstract/particle_holder/holder_right
 
+	///linked mech computer
+	var/obj/machinery/computer/mech_builder/linked_builder
+
```

Comment:
None of these link vars need to exist

Reply:
i can see the linked_mech being useless, its an oversight of mine, but i think linked builder does need to exist, how can the mech unlock the computer after being destroyed if it doesnt know which computer made it at all?

Reply:
removed linked_mech because it was not needed, but linked_builder is still there so the mech can unlock the computer after dying